ebu_tt_live/config/node.py

```diff
@@ -192,6 +192,20 @@ def __init__(self, config, local_config):
         self._create_input(config)
         self._create_output(config)
 
+class Denester(ConsumerMixin, ProducerMixin, NodeBase):
+    required_config = Namespace()
+
+    def _create_component(self, config):
+        self.component = processing_node.DenesterNode(
+            node_id=self.config.id
+        )
+
+    def __init__(self, config, local_config):
+        super(RetimingDelay, self).__init__(config, local_config)
```

Comment:
This should be `Denester` instead of `RetimingDelay` ;-)

Reply:
Yikes! Good spot, thanks for that. Weird python, I wonder what it actually does when we write it like this?!

Reply:
Python then aborts with a type error:

    TypeError: super(type, obj): obj must be an instance or subtype of type

Reply:
Oh I see, fair enough. Weird that we never hit that bug.